src/lib/Behat/Hooks.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace EzSystems\EzPlatformAdminUi\Behat;
+
+use Behat\MinkExtension\Context\RawMinkContext;
+use Behat\Symfony2Extension\Context\KernelDictionary;
+
+class Hooks extends RawMinkContext
+{
+    use KernelDictionary;
+
+    /** @BeforeScenario
+     */
+    public function restartSessionBeforeScenario()
```

Comment:
I had to add, otherwise browser session was not restarted after restoring platform-ee-demo database. Not sure why 🤔